### ISSUE TITLE
libc updated its appveyor

### ISSUE THREAD

**Issue by BurntSushi** [japaric/rust-everywhere#58]
I've been experiencing trouble with appveyor for the past day or so, where the build downloads Rust, hangs for an _hour_ and then gives up. You can see an example here: https://ci.appveyor.com/project/BurntSushi/ripgrep/build/1.0.21 --- I don't think anything changed on my end to cause this.

In any case, I went searching to see how our `libc` crate does things and found that they've updated to using `rustup-init.exe`: https://github.com/rust-lang/libc/commit/92105c645d93e1277f0dfb05f26785dd7f5b5ccd#diff-180360612c6b8c4ed830919bbb4dd459

Once I switched to `rustup-init.exe`, the Windows build started working again: https://ci.appveyor.com/project/BurntSushi/ripgrep/build/1.0.24

I'm afraid my Windows-fu is pretty weak, so I don't really know what the underlying cause was.

**Comment by japaric** [comment]
> I've been experiencing trouble with appveyor for the past day or so, where the build downloads Rust, hangs for an hour and then gives up. 

:crying_cat_face: 

> You can see an example here:

That looks ... weird. Perhaps static.r-l.o changed something ssl related. I've seen Start-FileDownload fail before due to that. The fix is call some other command to [enable certain types of ssl/tls](https://github.com/japaric/smoke/blob/46b90b7948ab8b82409357e44a0d5488eb02641f/ci/install.ps1#L17) that are disabled by default.

> In any case, I went searching to see how our libc crate does things and found that they've updated to using rustup-init.exe

We have updated our appveyor.yml to use rustup as well. In #57.

> Once I switched to rustup-init.exe, the Windows build started working again

Glad to hear it's working again. :+1:

I don't think there's left that's actionable about this issue since we are also using rustup in our appveyor.yml so I'm going to close this. I do think there is something that we coud do better: we could have some way to notify users (probably, in the templates themselves) that the .yml templates have been updated; I'll open an issue about that.
